vnext/ReactWindowsCore/Modules/PlatformConstantsModule.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "PlatformConstantsModule.h"
+#include <VersionHelpers.h>
+
+using Method = facebook::xplat::module::CxxModule::Method;
+
+namespace facebook::react {
+
+const char *PlatformConstantsModule::name = "PlatformConstants";
+
+std::string PlatformConstantsModule::getName() {
+  return PlatformConstantsModule::name;
+}
+
+std::vector<Method> PlatformConstantsModule::getMethods() {
+  return {};
+}
+
+std::map<std::string, folly::dynamic> PlatformConstantsModule::getConstants() {
+  return {
+    // We don't currently treat Native code differently in a test environment
+    {"isTesting", false},
+
+    // Since we're out-of-tree, we don't know the exact version of React Native
+    // we're paired with. Provide something sane for now, and try to provide a
+    // better source of truth later. Tracked by Issue #4073
+    {"reactNativeVersion", folly::dynamic::object("major", 0)("minor", 61)("patch", 0)}
```

Comment:
We shouldn't be able to let the major or minor versions go out of sync with reality, since that would cause a CI break from the version check.

I would rather just not add the patch version at all until we have something reliable, but there is code that will try to format it.

Reply:
Maybe we should do something similar to what we do with RNW_PKG_VERSION_STR to stamp the dll version number

Reply:
Discussed offline: I'm going to try C++ code version stamping upstream, since they do source-code version stamping already for other platforms.